BuoyTest/src/actions/Cocktails.js

```diff
@@ -0,0 +1,12 @@
+import {
+  LOAD_COCKTAILS,
+  SELECT_COCKTAIL
+} from './ActionTypes';
+
+export const loadCocktails = (cocktails) => (dispatch) => {
```

Comment:
At EQX, the action functions generally return just the action object (don't dispatch). The dispatch is executed in the caller

Reply:
Updated!